neo/Prompt/Commands/Invoke.py

```diff
@@ -309,7 +315,7 @@ def test_invoke(script, wallet, outputs, withdrawal_tx=None,
         wallet_tx = tx
     else:
         try:
-            wallet_tx = wallet.MakeTransaction(tx=tx, from_addr=from_addr)
+            wallet_tx = wallet.MakeTransaction(tx=tx, from_addr=from_addr, skip_fee_calc=True)
```

Comment:
if we skip fee calculations, will we then not get wrong information in places that print information like this?
https://github.com/CityOfZion/neo-python/blob/e561e5b57d4093a9ee0c87a8312d4a4d16f6fd5e/neo/Prompt/Commands/SC.py#L188-L189

Reply:
In this case, no because I include the logic after the transaction is completely constructed here https://github.com/CityOfZion/neo-python/pull/869/files/641ec3a954f5694147061ea58e45d4756f134690#diff-5ee74e6bf4c83897694f97ae0557af8eR388
Also, if the exception is raised here then Python will fail here https://github.com/CityOfZion/neo-python/pull/869/files/641ec3a954f5694147061ea58e45d4756f134690#diff-5ee74e6bf4c83897694f97ae0557af8eR322
because `wallet_tx` will have been called before being defined.